### ISSUE TITLE
Command line noise when running directly from the EXE

### ISSUE THREAD

**Issue by stretchtiberius** [atom/atom#11686]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

When using the portable (zip) version of atom in Windows, a bunch of errors are output to the command line when opening and closing atom.

Here are the errors when opening Atom:
```
[1548:7660:0503/201421:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=http://54.249.141.255:1127/post --application-name=Atom --v=1]
[1548:7660:0503/201421:VERBOSE1:crash_service.cc(142)] window handle is 000402AE
[1548:7660:0503/201421:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\Atom Crash Service
dumps at C:\Users\ADMINI~1\AppData\Local\Temp\Atom Crashes
[1548:7660:0503/201421:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\ADMINI~1\AppData\Local\Temp\Atom Crashes\crash_checkpoint.txt
server is http://54.249.141.255:1127/post
maximum 128 reports/day
reporter is electron-crash-service
[1548:7660:0503/201421:VERBOSE1:crash_service_main.cc(84)] Ready to process crash requests
[1548:4968:0503/201421:VERBOSE1:crash_service.cc(323)] client start. pid = 4232
App load time: 230ms
[1548:4968:0503/201421:VERBOSE1:crash_service.cc(323)] client start. pid = 7396
[2888:5712:0503/201421:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=http://54.249.141.255:1127/post --application-name=Atom --v=1]
[2888:5712:0503/201421:VERBOSE1:crash_service.cc(142)] window handle is 0013021E
[2888:5712:0503/201421:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\Atom Crash Service
dumps at C:\Users\ADMINI~1\AppData\Local\Temp\Atom Crashes
[2888:5712:0503/201421:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\ADMINI~1\AppData\Local\Temp\Atom Crashes\crash_checkpoint.txt
server is http://54.249.141.255:1127/post
maximum 128 reports/day
reporter is electron-crash-service
[2888:5712:0503/201421:ERROR:crash_service.cc(301)] could not start dumper
```

Here are the errors after closing Atom:
```
[1548:4368:0503/201432:VERBOSE1:crash_service.cc(331)] client end. pid = 7396
[1548:4368:0503/201432:VERBOSE1:crash_service.cc(331)] client end. pid = 4232
[1548:4368:0503/201433:VERBOSE1:crash_service.cc(352)] zero clients. exiting
[1548:7660:0503/201433:VERBOSE1:crash_service.cc(490)] session ending..
[1548:7660:0503/201433:VERBOSE1:crash_service.cc(495)] clients connected :2
clients terminated :2
dumps serviced :0
dumps reported :0
[1548:7660:0503/201433:VERBOSE1:crash_service_main.cc(89)] Session end. return code is 0
```

### Steps to Reproduce

1. Download atom-windows.zip version 1.7.3 and extract the files to a folder.
2. Open the command line and do one of the following:
    1. Change to the directory where Atom was extracted.
    2. Append the directory where Atom was extracted to the PATH variable.
3. From the command line, run ```atom.exe```

**Expected behavior:** Atom should load without outputting any message to the command line.

**Actual behavior:** A bunch of error messages are output to the command line.

### Versions
```
Atom    : 1.7.3
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```
I believe the problem exists for all of the 1.7 versions.

```
'apm' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by 50Wliu** [comment]
All this log output is expected, though I agree it would be much better if it wasn't shown.

(Also, your app load time...how I wish my Atom could load that fast :disappointed:)

**Comment by stretchtiberius** [comment]
This output is not produced when using the Atom installer. Only, when using the zip file. Shouldn't both distributions have the same command-line output?

**Comment by 50Wliu** [comment]
Huh, I get the same output, but maybe that's because I'm building from source.

**Comment by lee-dohm** [comment]
Doesn't the installed version run something other than `atom.exe` first? Like `atom.cmd` somewhere?

**Comment by damieng** [comment]
I also get this output spew running dev builds...  It's not an error but it sure is annoying.

**Comment by darkmorpher** [comment]
Isn't there a -silent switch?

**Comment by damieng** [comment]
I don't think there is a -silent switch and I'm not sure that's what we want here.

Basically we are getting a bunch of unexpected warning errors that the Mac version doesn't emit. We either need to fix the underlying thing causing these warnings or turn up the output level to exclude them - I think we still want other output.

**Comment by stretchtiberius** [comment]
Like the Mac version, the Windows version from the installer does not emit these warning errors.

Does the installer setup somehow eliminate these warnings in an extra step not performed on the portable version or when compiled from source?

**Comment by damieng** [comment]
The Windows version from the installer 1.7.3 does actually emit these warnings if you run `atom.exe` directly - normally however you just type `atom` which runs `atom.cmd` which will either:

1. Run atom.js via node.exe if no args specified
2. Run atom.exe in a background PowerShell if --wait specified
3. Run atom.exe directly if an argument specified (e.g. `atom --foreground`)

**Comment by damieng** [comment]
So this didn't happen in Atom 1.6.1 but did occur in Atom 1.7.0-beta1...

**Comment by damieng** [comment]
Working on this in the branch https://github.com/atom/atom/tree/command-line-noise-cleanup

Appear to be three issues here:

1. Crash reporter started twice - should only do that on Mac OS X 
This was the cause of 'could not start dumper' on Windows even when Atom *not* currently running

2. Crash reporter starts before App checks if it is already running
This is the cause of 'could not start dumper' on Windows when Atom *is* already running

3. VERBOSE1 messages
Electron logging level is not set so we're getting everything

**Comment by damieng** [comment]
Item 1 was fixed in df22e0bc5b362647344e404d4427e461e3858825

**Comment by damieng** [comment]
Item 3 is actually an issue in Electron that is fixed in 1.2.1 there. I have no ETA on when Atom would be using that version.

**Comment by damieng** [comment]
We have upgraded Electron to 1.3.5 over the past week or two and I can happily report that the crash reporter noise is gone.  The only output right now is 'App load time'.
